packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -109,7 +325,26 @@ export class WcSdk {
         return await WcSdk.sendRequest(this.wcClient, this.session, this.chainId, request)
     }
 
-    async invokeFunction(request: ContractInvocation) {
+    /**
+     * Sends an 'invokefunction' request to the Wallet and it will communicate with the blockchain. It will consume gas and persist data to the blockchain.

```

Comment:
```suggestion
     * Sends an 'invokefunction' request to the Wallet, which should approve it and communicate the transaction to the blockchain. It will consume gas and persist data to the blockchain.

```

_similar doubt as before. It's unclear who's talking to the blockchain. I'm taking it's the wallet._